tests/integration/helpers.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+async def get_unit_address(ops_test, app_name: str, unit_num: int) -> str:
+    status = await ops_test.model.get_status()  # noqa: F821
```

Comment:
Missing docstring.

Reply:
In all our repos we do not check for docstrings
```
# D100, D101, D102, D103: Ignore missing docstrings in tests
per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,C801"]
```
But in some case docstrings are a good idea anyway.
However, in this particular case the docstring would be identical to the function name.

Reply:
I think it is good practice to always have a docstring, regardless of the function, public, private etc. I do not think the docstring will be same as the function name, for example 
https://github.com/canonical/prometheus-k8s-operator/blob/main/tests/integration/helpers.py